docs/EN_US/ECLStandardLibraryReference/SLR-Mods/MonthsBetween.xml

```diff
@@ -30,6 +30,16 @@
           <entry>The last date value in Date_t format.</entry>
         </row>
 
+        <row>
+          <entry><emphasis>month_ends_equal</emphasis></entry>
+
+          <entry>Optional. If TRUE and both dates fall on the last day of
+          their respective months difference between the dates will be treated
```

Comment:
I think "... respective months difference ..." should be "... respective months, the difference ..."

Reply:
Edits made